package/src/hydra-ai/ai-service.ts

```diff
@@ -156,12 +199,15 @@ ${
     : `You can also use any of the provided tools to fetch data needed to pass into the component.`
 }
 
-${this.generateZodTypePrompt(schema)}`;
+${this.generateZodTypePrompt(generateSpecifiedComponentSchema)}`;
```

Comment:
If we specify the response schema in the openAI call now, I guess we can remove this part of the prompt. Also I think this is the only place the `generateZodTypePrompt` function is used, so we could probably remove that too